bridge_ui/src/hooks/useFetchTargetAsset.ts

```diff
@@ -114,7 +109,7 @@ function useFetchTargetAsset(nft?: boolean) {
     ]
   );
   useEffect(() => {
-    if (isRecovery || argsMatchLastSuccess) {
```

Comment:
For some context - the targetAsset isn't set in the redux store when the user is navigated to the redeem step (4) on the Transfer page during the recovery workflow. This hook will now fire after they're navigated to the redeem step and set the targetAsset.